tests/manual/progress/test-analyze-complexity.js

```
@@ -0,0 +1,693 @@
+				await testNonStreaming();
+				break;
+
+			case 'both':
+				console.log(
+					chalk.yellow(
+						'Running both MCP streaming and non-streaming tests...\n'
+					)
+				);
+				const mcpStreamingResult = await testMCPStreaming();
+				console.log('\n' + '='.repeat(60) + '\n');
+				const nonStreamingResult = await testNonStreaming();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)